<ui version="4.0" >
 <class>ViewEditor</class>
 <widget class="QDialog" name="ViewEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>View Editor</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Presets</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QComboBox" name="presets_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="currentIndex" >
         <number>-1</number>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="use_" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text" >
           <string>Use</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="update_" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text" >
           <string>Update</string>
          </property>
          <property name="autoDefault" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text" >
           <string>Delete</string>
          </property>
          <property name="autoDefault" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Frequency Scale (X Axis)</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Min:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="xMin_" >
        <property name="toolTip" >
         <string>Max viewable range</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <double>-999.990000000000009</double>
        </property>
        <property name="maximum" >
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep" >
         <double>10.000000000000000</double>
        </property>
        <property name="value" >
         <double>-500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="xMinScale_" >
        <property name="currentIndex" >
         <number>2</number>
        </property>
        <item>
         <property name="text" >
          <string>Hz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>kHz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>MHz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>GHz</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Max:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="xMax_" >
        <property name="toolTip" >
         <string>Max viewable range</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <double>-999.990000000000009</double>
        </property>
        <property name="maximum" >
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep" >
         <double>10.000000000000000</double>
        </property>
        <property name="value" >
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="xMaxScale_" >
        <property name="currentIndex" >
         <number>2</number>
        </property>
        <item>
         <property name="text" >
          <string>Hz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>kHz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>MHz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>GHz</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7" >
     <property name="title" >
      <string>db Scale (Y Axis)</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Min:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="yMin_" >
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>200.000000000000000</double>
        </property>
        <property name="value" >
         <double>-120.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Max:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="yMax_" >
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>200.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="revert_" >
       <property name="text" >
        <string>Revert</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply_" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>presets_</tabstop>
  <tabstop>use_</tabstop>
  <tabstop>update_</tabstop>
  <tabstop>delete_</tabstop>
  <tabstop>xMin_</tabstop>
  <tabstop>xMinScale_</tabstop>
  <tabstop>xMax_</tabstop>
  <tabstop>xMaxScale_</tabstop>
  <tabstop>yMin_</tabstop>
  <tabstop>yMax_</tabstop>
  <tabstop>revert_</tabstop>
  <tabstop>apply_</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
